README.md

```diff
@@ -124,15 +124,15 @@ Octant should immediately launch your default web browser on `127.0.0.1:7777`.
 Or to run it on a specific host and fixed port:
 `$ OCTANT_LISTENER_ADDR=0.0.0.0:8900 octant`
 
-For configuring Octant, setting up a development environment, or running tests, refer to the documentation [here](site/docs/master/getting-started.md).
+For configuring Octant, setting up a development environment, or running tests, refer to the documentation [here](https://octant.dev/docs/master).
```

Comment:
can we just make documentation the link?

Reply:
Good call! Fixed.